src/views/OrdersView.vue

```diff
@@ -66,6 +91,10 @@ export default Vue.extend({
     }
   },
   methods: {
+    deleteOrderConfirmed: function (orderNumber) {
+      this.deleteOrder(orderNumber)
```

Comment:
what is delete order fails

Reply:
That would trigger an exception in the console without notice. Should I handle this case?